js/buttons.js

```diff
@@ -0,0 +1,111 @@
+const home_button = document.getElementById("iphone-home")
+const mute_button = document.getElementById("iphone-mute")
+const volume_up_button = document.getElementById("iphone-v-up")
+const volume_down_button = document.getElementById("iphone-v-down")
+const power_button = document.getElementById("iphone-power")
+
+// initial state of iPhone:
+const iphone_state = {
+    screen: true,
+    block: true,
+    volume: 5,
+    mute: false,
+}
+
+// Variables that are needed to clear setTimeout when the buttons are pressed
+let hide_volume;
+let hide_mute;
+
+
+// Home Button Listener
+home_button.addEventListener("click", ()=>{
+    if (iphone_state.screen){
+        iphone_state.block = false;
+
+    } else{
+        iphone_state.screen = true;
+        iphone_state.block = false;
+
+        document.getElementById('iphone-screen').style.opacity = "1";
+    }
+
+    document.getElementById('iphone-lock-img').src = "../img/lock-open.svg"
```

Comment:
This and other images have wrong paths. It doesn't show me these images unless I remove the `../` part. Your JS file is in the JS folder, but you import it into your index.html which is on the root level.

Reply:
Thanks for the comment, I didn't notice this error! I'll fix it!

For some reason, all the pictures worked for me when I checked on different browsers.